CONTRIBUTING.md

```diff
@@ -0,0 +1,296 @@
+> You are reviewing the contributing document for getkong.org. If you want to know how to contribute to Kong itself, then please go [here](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md)
+
+# Contributing to getkong.org 📜 🦍
+
+Hello, and welcome! Whether you are looking for help, trying to report a bug,
+thinking about getting involved in the project or about to submit a patch, this
+document is for you! Its intent is to be both an entry point for newcomers to
+the community (of all technical backgrounds), and a guide/reference for
+contributors and maintainers.
+
+Consult the Table of Contents below, and jump to the desired section.
+
+## Table of Contents
+
+- [Where to seek for help?](#where-to-seek-for-help)
+  - [Enterprise Edition](#enterprise-edition)
+  - [Community Edition](#community-edition)
+- [Where to report bugs?](#where-to-report-bugs)
```

Comment:
I'm not sure if the above sections are necessary? For someone who just wants to ix a typo in the docs, they might add quite a lot of overhead and the contributor might just skip reading the whole doc? It will also be a maintenance burden for us to sync it with the Kong one.

I like the above note (you are viewing the getkong.org contributing guide, and not Kong), so maybe we can do:

```
## Where to seek for help?

See the [Kong Contributing Guideline](kong/kong.contributing.md#where-to-seek) for an overview of the channels at your disposal.

## Where to report bugs?

If the bug is about the getkong.org website, please report it in this [repository's issues tracker](getkong.org/issues/new). If the bug is related to Kong itself, please  see the [Kong Contributing Guideline](kong/kong.contributing.md#where-to-report-bugs).

## Contributing

We welcome all kings of contributions, not only to Kong, but also to its documentation! If you wish to contribute to the documentation (by fixing a typo of making any other kind of improvement), please read the below section about [submitting a
+patch](#submitting-a-patch).

If you wish to contribute to Kong itself, see the [Kong Contributing Guideline](kong/kong.contributing.md#contributing).
```

Etc... The goal being to not duplicate content between the two contributing, except for the commit format which is slightly different